utk/utk.go

```diff
@@ -218,6 +218,9 @@ func (a *assembleCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface
 		return subcommands.ExitFailure
 	}
 	jsonbuf, err := ioutil.ReadFile("summary.json")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
do a log.Print and a subcommands.ExitFailure instead

Reply:
Address is #76